spec/models/customer_item_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe CustomerItem, type: :model do
+  it "has a valid relationship to customer" do
```

Comment:
You can just use shoulda-matchers here to test relationships and that will suffice!!

`it { should belong_to :customer }`
`it {should belong_to :item }`

Reply:
Under the hood, these 2 lines are testing all of what you wrote here. It's much easier to stick to these nifty shoulda-matchers!